tunnel_test.go

```diff
@@ -255,6 +255,9 @@ func TestSingleLatencyRequest(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// wait til the environment is ready, just for test
+	time.Sleep(time.Second * 2)
```

Comment:
this will create problems eventually, we can find a better way to handle synchronization later